### ISSUE TITLE
Rearing/specimen guide

### ISSUE THREAD

**Issue by Megachile** [jeffdc/gallformers#262]
We should create a section of the site that collects useful information on what to do with specimens in different inducer taxa, ie how to collect and store fungal or mite specimens, how to rear cecidomyiid or cynipid adults, etc, linking existing resources or reprinting where possible, synthesizing and adding our own text and images, etc.

**Comment by jeffdc** [comment]
Along these same lines I was thinking it would be nice to have a section on gall finding like the post that Tim made on iNat. 

I think that what we are after is something like a blog/CMS (Content Management System) where we can post text and images that are topical but are not stored in the database with the main gall information. The good news is that the main frameweork I am using (next.js) is perfect for this. 

The easiest way is to add a section for creating posts to the admin toolset. From here a new post could be created. The post would simply be a title, timestamp, author, and content in Markdown form. I would repurpose the image upload stuff we are already have to allow uploading of images. This is not a full blown CMS but it would probably work.

Another option would be to evaluate any CMS libraries that run on top of next.js and add that. It is probably overkill for what we want.

If this is inline with your thinking I can look into options in more detail.

**Comment by Megachile** [comment]
That makes sense to me, though I'd want to display them on the public side in some way other than a chronological blogroll style if possible.

**Comment by jeffdc** [comment]
Agreed. Would probably need to make a way to add things to the Resources menu.

**Comment by NancyAsquith** [comment]
A good basic start would be to add a simple-text page [for now] to "Resources" and provide a link to Charley Eiseman's rearing instructions at https://bugtracks.wordpress.com/rearing/  and a sentence on how to preserve adults. Sections on Why rear?, Rearing different categories of gallformers, Success stories, and Cautionary tales can be added later. 

Then start a topic on the iNat Forum, encouraging people to provide stories, add tips, and ask questions. Yann, in a comment to https://www.inaturalist.org/observations/94667369, also mentioned a comment from the iNat Discord
